app/src/main/java/org/openobservatory/netprobe/utils/Alert.java

```diff
@@ -52,4 +52,57 @@ public void onClick(DialogInterface dialog, int id) {
 
                 }).show();
     }
+
+    public static String formatString(String text){
```

Comment:
Uhm, this is to pretty-indent the JSON right? I'd see if it's possible to do this task using `Gson`.